src/content/structured/components/tree-view/guidance.mdx

```diff
@@ -9,17 +9,22 @@ title: "Tree view"
 
 status: "CANARY"
 
-subTitle: "Use the tree view component to display hierarchical data in a tree structure."
+subTitle: "Tree view provides a navigation tree for use in applications with hierarchical organised pages."
```

Comment:
I think the subtitles on the Code and Accessibility pages should be updated too so that they match